index.js

```diff
@@ -181,23 +181,33 @@ express()
   .get('/addCustom', (req, res) => res.render('pages/addCustom'))
   .post('/addCustom', jsonParser, async function(req, res) {
     try {
+      //var userID = userProfile ? userProfile.id : 1; 
+      var userID = '1'
```

Comment:
@mdschnei I had this line in here for testing purposes, should it be put back to the commented out line?

Reply:
We should keep the variable one (var userID = userProfile ? userProfile.id : 1;) since that one will correctly keep track of which user is logged in (assuming it is working correctly).

Hard coding userID as 1 will cause every new custom product to be added to _only_ the default user's list of available products, and also will add it only to the default user's fridge, even if a different user is currently logged in at the time.

Reply:
Fixed!